wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/Views/Components/DeviceDetailsButtonsView.swift

```diff
@@ -83,8 +83,6 @@ struct DeviceDetailsButtonsView: View {
                     Divider()
                     getCertificateButton.padding()
                 }
-                Divider()
-                showCertificateButton.padding()
```

Comment:
https://www.figma.com/file/gSmh7BRyq9SGKkyOLSbOdM/E2E-Identity?type=design&node-id=234-34282&mode=design&t=I2XpMNbwg4u0aFxZ-0

Based on this UI we have to show Show certificate button

Reply:
It's a wrong UI. You don't have a certificate if the status is `Not Activated`. What should show? 
The correct design is for Web